### PR TITLE
feat: added card component page and css file

### PR DESCRIPTION
**Pull request by zabihshaik**
Added Card Component Page and imported it's css file in main.css.
The Card component has 6 Variants:

- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards
- Cards with shadow

The Card page looks like:

Card Page

Deploy Preview Link:
https://deploy-preview-5--neoz-ui.netlify.app/pages/card.html

### CODE REVIEW

File: styles/card.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    background-color: var(--bg-body);
+    cursor: pointer;
+}
+
+.card-icons {
+    display: flex;
+    justify-content: flex-end;
+    gap: 2.5rem;
+    margin-right: 1.5rem;
+    margin-bottom: 1rem;
```
Comment (yashpatel1797) [inline styles/card.css]: You can use shorthand properties here.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: styles/card.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    justify-content: space-around;
+    margin: 0.5rem 1rem 1rem 1rem;
+    padding: 0.3rem;
+    cursor: pointer;
+}
+
+.card-btn {
+    text-decoration: none;
+    background: transparent;
+    border-radius: 10px;
```
Comment (NeerajVodala) [inline styles/card.css:64]: be consistent with units

File: pages/card.html [hunk 3]
```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Zabihulla Shaik" />
```
Comment (parvez-rashid) [inline pages/card.html:8]: Great that you've used meta tags for SEO.

File: styles/card.css (hunk 1 above)
Comment (zabihshaik) [reply styles/card.css]: Yeah. Will update it. Thanks for pointing out.

**Review by sanu29 — APPROVED** [review]
Good to merge

File: styles/card.css (hunk 2 above)
Comment (zabihshaik) [reply styles/card.css:64]: Yes. Actually, I have not converted px less than or equal to 10px to rem as per Vikram.

**Review by NeerajVodala — APPROVED** [review]
Be consistent with units. Rest everything looks good to me.
All the Best

File: pages/card.html [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+
+
+            <h3 class="heading-small">Card with Text Overlay</h3>
+            <p class="description">Cards With Badges are similar to basic cards, but with badges added to them. You can
```
Comment (parvez-rashid) [inline pages/card.html]: I think here you're supposed to give description for card with text overlay.

**Review by parvez-rashid — APPROVED** [review]
I've added my comments.  The UI looks great. All the best!

File: pages/card.html (hunk 4 above)
Comment (zabihshaik) [reply pages/card.html]: Yeah. Will modify it. Thanks for pointing it out Parvez.

File: styles/card.css (hunk 2 above)
Comment (NeerajVodala) [reply styles/card.css:64]: Oh, didn't know that. I guess I missed that part(or session).

**Comment by zabihshaik** [comment]
Fixed the Changes as per the PR Reviews.

**Pull request merged**